docs/src/installing.md

```diff
@@ -40,17 +40,17 @@ Build and install the tool via cargo...
 
 ## Build From Source
 
-Debug build with logging enabled:
```

Comment:
Hm, this reads as if you have to set it for building, not at run time; unless that really is the case, I think the previous note was better

Reply:
Fair point. What about the fixupped version?

Reply:
Looks good